foundation/foundation-mda/src/test/java/com/boozallen/aiops/mda/metamodel/element/PipelineSteps.java

```diff
@@ -594,6 +625,7 @@ public void aPipelineIsConfiguredForMessagingWithoutAChannelName() throws Except
     }
 
     @When("machine learning pipelines are validated")
+    @When("the pipelines are validated")
```

Comment:
Q: Should this method have both @When annotations on it?

Reply:
Initially I was reusing the test step logic with different wording, but I ended up moving it to its own method.